wercker.yml

```diff
@@ -42,28 +42,37 @@ build:
 
 deploy:
   steps:
+    - add-to-known_hosts:
+        hostname: github.com
+    - add-ssh-key:
+        keyname: GITHUB_KEY
     - script:
-        name: Configure git
+        name: config git
         code: |
           git config --global user.email "devteam@technologyadvice.com"
           git config --global user.name "TechnologyAdvice"
-
-          # remove current .git folder
-          rm -rf .git
```

Comment:
The Wercker docs say to nuke the whole git repo, then init a new one, and push it.  We aren't doing that craziness anymore.